docker-compose.yml

```diff
@@ -23,7 +23,8 @@ services:
       TZ: "${HOST_TZ}" # Time Zone
       GW_ID: "${ID}"
       SIP_ACCOUNT: "${ACCOUNT}" # <sip:user@domain;transport=tcp>;auth_pass=pass;answermode=auto;medianat=turn;stunserver=turn:turnserver.org:3478;stunuser=username;stunpass=passwd
-      BROWSE_FILE: "rendezvousjitsi.py" # python browsing based on Selenium
+      BROWSE_FILE: "jitsi.py" # python browsing based on Selenium
+      WEBRTC_DOMAIN: "rendez-vous.renater.fr" # WebRTC domain name (only for Jitsi web conferences)
```

Comment:
This parameter should not be variabilized (with something like `"${JITSI_FQDN}"`, and added in the `sipmediagw.cfg` file) in order not to have to overwrite the docker-compose (and only the `sipmediagw.cfg` file) for the whole configuration?

Reply:
Good idea, BROWSE_FILE and WEBRTC_DOMAIN could be set from environment variables, assigned in SIPMediaGW.sh and coming from sipmediagw.cfg
Another point is that WEBRTC_DOMAIN in the target use case could be obtained by requesting a conference mapper:
https://rendez-vous.renater.fr/conf-api/conferenceMapper?conference=testmapping for instance
Thus, finally WEBRTC_DOMAIN could be replaced by CONFMAPPER_PATH